dist/schema.json

```diff
@@ -13145,9 +13165,25 @@
               "follow_request",
               "favourite",
               "poll",
-              "update",
-              "admin.sign_up",
-              "admin.report"
+              "update"
+            ]
+          },
+          "alerts[admin": {
```

Comment:
The `poll` updates look good.


`alerts[admin.sign_up]` is not being parsed correct though. `admin.sign_up` should not be split and should remain the name of the property.

Also add tests for the changes.